### ISSUE TITLE
Unite causes vim to freeze with 100% cpu usage

### ISSUE THREAD

**Issue by jakubgs** [Shougo/unite.vim#798]
Hi,

I'm using vim 7.4.488 on Debian Jessie with newest Unite and vimproc.vim(for file_rec/async). I've managed to reproduce this problem without vimproc.vim so I don't think it's relevant.

At (seemingly)random times vim will freeze, and when looked up in htop it's using 100% or close to 100% cpu:

![gvim_100_cpu](https://cloud.githubusercontent.com/assets/2212681/5453017/4306ca72-8522-11e4-8f4f-65211ba8cb01.png)

The freezes sometimes happen when actively using Unite, but sometimes not.

I've tried tracking down this problem by using :profile commands but nothing shows up. Execution times logged using :profile seem to not include the long freezes(several seconds, more than a minute, sometimes endless).

Here's a profile.log from a freeze that lasted about more than 3 minutes:

https://gist.githubusercontent.com/PonderingGrower/4689817e5bcfbf52895c/raw/7ab81ae22d774cffd9551274bc19a02c59c7b618/profile.log

In order to reproduce it I've spent a bit of time doing searches with unite. To be exact using this:

```
nnoremap <c-i>     :execute('Unite buffer file_mru file_rec/async:'.g:projectroot)<CR>
```

I have tried disabling plugins one by one and I've narrowed down this issue to Unite, but I do not know what to do next to identify what exactly is causing it.

I will be happy to follow any debugging steps you might suggest.

**Comment by Shougo** [comment]
Fixed in Vim 7.4.615

https://groups.google.com/forum/#!topic/vim_dev/pXjj7rww5RY

**Comment by PonderingGrower** [comment]
That's awesome to hear! I won't have to keep building my own deb package.

Thansk Shougo, you are the man!
